apps/src/AzureTextToSpeech.js

```diff
@@ -10,11 +10,17 @@ import Sounds from '@cdo/apps/Sounds';
  * @param {Array<string>} profaneWords Any profanity in the response. Used to determine whether the response should be cached and played.
  * @param {Error} error Any error that occurs while requesting the sound or checking for profanity.
  */
-export class SoundResponse {
-  constructor(id, bytes, playbackOptions, profaneWords = [], error = null) {
+class SoundResponse {
+  constructor(id, bytes, profaneWords = [], error = null, onEnded = null) {
     this.id = id;
     this.bytes = bytes;
-    this.playbackOptions = playbackOptions;
+    this.playbackOptions = {
+      volume: 1.0,
+      loop: false,
+      forceHTML5: false,
+      allowHTML5Mobile: true,
+      onEnded
+    };
```

Comment:
should this be an optional constructor param? i originally had this in AzureTextToSpeech because it doesn't feel like something SoundResponse should necessarily dictate. we could then have the default options we have here, and spread any overrides that come from the constructor param